apps/demos/src/zone/event-names.ts

```diff
@@ -0,0 +1,120 @@
+// Standard Events
+
+// User Interface Events
+
+// UIEvent
+export const uiEvent = [
+  'load',
+  'unload',
+  'abort',
+  'error',
+  'select'
+];
+
+// Focus Events
+export const focusEvent = [
+  'blur',
+  'focus',
+  'focusin',
+  'focusout'
+];
+
+// Mouse Events
+// (MouseEvent)[https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent]
+export const mouseEvent: (keyof WindowEventMap)[] = [
+  'mousedown',
+  'dblclick',
+  'mouseenter',
+  'mouseleave',
+  'mousemove',
+  'mouseout',
+  'mouseover',
+  'mouseup',
+  'click'
+];
+
+// Wheel Events
+export const wheelEvent: (keyof WindowEventMap)[] = [
+  // (WheelEvent)[https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent]
+  'wheel'
+];
+
+// Input Events
+export const inputEvent: (keyof WindowEventMap)[] = [
+  // 'beforeinput',
+  'input'
+];
+
+// Keyboard Events
+export const keyboardEvent: (keyof WindowEventMap)[] = [
+  'keydown',
+  'keyup'
+];
+
+// Composition Events
+export const compositionEvent = [
+  'compositionstart',
+  'compositionupdate',
+  'compositionend'
+];
+
+export const touchEvents: (keyof WindowEventMap)[] = [
+  // [PointerEvent](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent)
+  'pointerover',
+  'pointerenter',
+  'pointerdown',
+  'pointermove',
+  // 'pointerrawupdate',
+  'pointerup',
+  'pointercancel',
+  'pointerout',
+  'pointerleave',
+  'gotpointercapture',
+  'lostpointercapture',
+  'pointerup',
+
+  // [TouchEvent](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent)
+  'touchstart',
+  'touchend',
+  'touchmove',
+  'touchcancel',
+
+  'drag',
+  'dragend',
+  'dragenter',
+  'dragleave',
+  'dragover',
+  'dragstart',
+  'drop'
+];
+
+export const formControlsEvents: (keyof WindowEventMap)[] = [
+  'change',
+  'blur',
+  'focus',
+  'contextmenu',
```

Comment:
is this only for formControls? i guess contextmenu is available on most elements isn't it?

Reply:
true! What should be the group?

Reply:
i'd say it's a global event. From what I can see in the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/contextmenu_event) the event exists on `Element`

Reply:
So I create one with subgroups